packages/wonder-blocks-icon/util/icon-paths.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+/**
+ * The source SVG paths for our icons at various sizes
+ */
+
+type IconPaths = {
+    [string]: {
+        small?: string,
+        medium?: string,
+        large?: string,
+        xlarge?: string,
```

Comment:
I worry about everything being optional.  Maybe we could say the type is:
```
({ small: string } | { medium: string } | { small: string, medium: string })
```
that way we guarantee that there's at least a `small` or `medium` size path.

Reply:
I guess this plan of ensure at least one item is present gets a little more complicated if design could provide us with assets for any one of the four sizes.